[update] Install Tobs (#1734)
timescale/docs: promscale/tobs/tobs.md

*  Install [Helm 3][helm] in your local machine.
* Install a certificate manager for your cluster.
   <highlight type="note">
   For the latest and updated instructions to install, see the

>    [certificate manager][cert-manager-docs] documentation. You may choose not to
>    install certificate manager if you plan to use tobs with opentelemetry support
>    disabled.

Reviewer: ```suggestion
   [certificate manager][cert-manager-docs] documentation. If you plan to use 
   tobs without OpenTelemetry support, you do not need to install certificate 
   manager.
```
